<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>licensePage</class>
 <widget class="QWidget" name="licensePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>536</height>
   </rect>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>531</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>567</width>
      <height>527</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="licenseYearLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseAgeGroupComboBox">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sukupuoli</string>
     </property>
    </widget>
    <widget class="QPushButton" name="licenseSavePushButton">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tallenna</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Eläin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Myönnetyt luvat</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="licenseSummaryTableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>521</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseGenderComboBox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="licenseAmountLineEdit">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Lupavuosi</string>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseAnimalComboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Ikäryhmä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Määrä</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>100</y>
       <width>130</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Lajittele</string>
     </property>
    </widget>
    <widget class="QComboBox" name="licenseSortComboBox">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
